string name
uint8 age
float32 weight
uint8 height
